apps/explorer/src/components/ui/TableCard.tsx

```diff
@@ -91,11 +92,18 @@ export function TableCard<DataType extends object>({
                                     columnKey={id}
                                     label={column.columnDef.header?.toString()}
                                     hasSort={column.columnDef.enableSorting}
-                                    onSortClick={
+                                    initSortOrder={TableHeaderCellSortOrder.Desc}
+                                    onSortClick={(key, sortOrder) => {
```

Comment:
if we click to sort by name and then by another category, I think only the icon of the last clicked category should change :pray: